language/German/strings.po

```diff
@@ -4565,1083 +4626,1083 @@ msgstr ""
+msgstr "Profiler anzeigen"
 
 #: src/emu/inpttype.ipp:913
 msgctxt "input-name"
 msgid "UI Toggle"
-msgstr ""
+msgstr "Benutzeroberfläche umschalten"
 
 #: src/emu/inpttype.ipp:914
 msgctxt "input-name"
 msgid "UI Release Pointer"
-msgstr ""
+msgstr "Benutzeroberfläche Zeiger Lösen"
 
 #: src/emu/inpttype.ipp:915
 msgctxt "input-name"
 msgid "UI Paste Text"
-msgstr ""
+msgstr "Benutzeroberfläche Text einfügen"
 
 #: src/emu/inpttype.ipp:916
 msgctxt "input-name"
 msgid "Save State"
-msgstr ""
+msgstr "Status speichern"
 
 #: src/emu/inpttype.ipp:917
 msgctxt "input-name"
 msgid "Load State"
-msgstr ""
+msgstr "Status laden"
 
 #: src/emu/inpttype.ipp:918
 msgctxt "input-name"
 msgid "UI (First) Tape Start"
-msgstr ""
+msgstr "Benutzeroberfläche (Erste) Kassette Start"
 
 #: src/emu/inpttype.ipp:919
 msgctxt "input-name"
 msgid "UI (First) Tape Stop"
-msgstr ""
+msgstr "Benutzeroberfläche (Erste) Kassette Stop"
 
 #: src/emu/inpttype.ipp:920
 msgctxt "input-name"
 msgid "UI External DAT View"
-msgstr ""
+msgstr "Benutzeroberfläche Externe DAT Anzeige"
 
 #: src/emu/inpttype.ipp:921
 msgctxt "input-name"
 msgid "UI Add/Remove favorite"
-msgstr ""
+msgstr "Benutzeroberfläche Favorit hinzufügen/entfernen"
 
 #: src/emu/inpttype.ipp:922
 msgctxt "input-name"
 msgid "UI Export List"
-msgstr ""
+msgstr "Benutzeroberfläche Liste Exportieren"
 
 #: src/emu/inpttype.ipp:923
 msgctxt "input-name"
 msgid "UI Audit Media"
-msgstr ""
+msgstr "Benutzeroberfläche Medien Überprüfen"
 
 #: src/osd/modules/input/input_windows.cpp:110
 #: src/osd/modules/input/input_sdlcommon.cpp:173
 msgctxt "input-name"
 msgid "Toggle Fullscreen"
-msgstr ""
+msgstr "Vollbild umschalten"
 
 #: src/osd/modules/input/input_windows.cpp:116
 msgctxt "input-name"
 msgid "Take Rendered Snapshot"
-msgstr ""
+msgstr "Gerenderten Schnappschuss machen"
 
 #: src/osd/modules/input/input_windows.cpp:127
 #: src/osd/modules/input/input_sdlcommon.cpp:221
 msgctxt "input-name"
 msgid "Record Rendered Video"
-msgstr ""
+msgstr "Gerendertes Video aufnehmen"
 
 #: src/osd/modules/input/input_windows.cpp:143
 msgctxt "input-name"
 msgid "Toggle Post-Processing"
-msgstr ""
+msgstr "Nachverarbeitung umschalten"
 
 #: src/osd/modules/input/input_sdlcommon.cpp:199
 msgctxt "input-name"
 msgid "Toggle Filter"
-msgstr ""
+msgstr "Filter umschalten"
 
 #: src/osd/modules/input/input_sdlcommon.cpp:205
 msgctxt "input-name"
 msgid "Decrease Prescaling"
-msgstr ""
+msgstr "Vorteilung erniedrigen"
 
 #: src/osd/modules/input/input_sdlcommon.cpp:215
 msgctxt "input-name"
 msgid "Increase Prescaling"
-msgstr ""
+msgstr "Vorteiler erhöhen"
 
 #: plugins/hiscore/init.lua:85
 msgctxt "plugin-hiscore"
 msgid "When updated"
-msgstr ""
+msgstr "Bei Änderung"
 
 #: plugins/hiscore/init.lua:85
 msgctxt "plugin-hiscore"
 msgid "On exit"
-msgstr ""
+msgstr "Beim Verlassen"
 
 #: plugins/hiscore/init.lua:86
 msgctxt "plugin-hiscore"
 msgid "Hiscore Support Options"
-msgstr ""
+msgstr "Highscore-Unterstützungs-Optionen"
```

Comment:
"Optionen für Highscoreunterstützung"? or "Highscoreunterstützungsoptionen"

Reply:
Changed to _Optionen für Highscore-Unterstützung_